test/metric_value_benchmark/eks_daemonset_test.go

```diff
@@ -66,19 +77,80 @@ func (e *EKSDaemonTestRunner) validateInstanceMetrics(name string) status.TestRe
 	return testResult
 }
 
-func (e *EKSDaemonTestRunner) GetTestName() string {
+func (e *EKSDaemonTestRunner) validateLogs(eks *environment.MetaData) status.TestResult {
+	testResult := status.TestResult{
+		Name:   "emf-logs",
+		Status: status.FAILED,
+	}
+
+	now := time.Now()
+	group := fmt.Sprintf("/aws/containerinsights/%s/performance", eks.EKSClusterName)
+
+	// need to get the instances used for the EKS cluster
+	EKSInstances, err := awsservice.GetEKSInstances(eks.EKSClusterName)
+	if err != nil {
+		log.Println("failed to get EKS instances")
```

Comment:
nit - would be nice to bubble up the error in the log:
```suggestion
		log.Println("failed to get EKS instances", err)
```

Reply:
done